add rasterio requirements & fix tests

waltaskew opened the pull request.

rsutormin reviewed (commented):
I think I fixed all these problems in https://github.com/UrbanSystemsLab/climateiq-cnn/pull/3 (for instance I switched to memory file in tests as we discussed there). Thanks for sharing this example though!

waltaskew commented:
> I think I fixed all these problems in #3 (for instance I switched to memory file in tests as we discussed there). Thanks for sharing this example though!

cool -- I'll close it
I made this as a PR into your branch, so if you'd wanted to, you could have merged this code directly into your branch.
